kerykeion/report.py

```
@@ -757,7 +762,7 @@ def _render_point_in_house_table(self, points: Sequence[PointInHouseModel], titl
             projected_house = f"{point.projected_house_number} ({point.projected_house_name})"
             table_data.append(
                 [
-                    f"{point.point_owner_name} – {point.point_name.replace('_', ' ')}",
+                    f"{point.point_owner_name} – {_humanize(point.point_name)}",
```

Ruff (0.15.10)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)